plugins/out_stackdriver/stackdriver.h

```diff
@@ -110,6 +110,19 @@ struct flb_stackdriver {
     flb_sds_t local_resource_id;
     flb_sds_t tag_prefix;
 
+    /* generic resources */
+    flb_sds_t location;
+    flb_sds_t namespace_id;
+    bool generic_resource_type;
```

Comment:
I found this naming a bit confusing, perhaps `is_generic_resource_type` is more clear? Up to you

Reply:
This was done mainly to match the same naming that was done for the k8s resources. However, using is_ would give a better idea of what type it is, so I will change both the generic_resource_type and k8s_resource_type to have the is_ prefix.